src/test/rustdoc-ui/doc-alias-same-name.rs

```diff
@@ -0,0 +1,8 @@
+// compile-flags: -Zdeduplicate-diagnostics=no
```

Comment:
Isn't this on by default for all UI tests?

Reply:
No it's not. We added it to ensure that the crate level attributes didn't show the errors twice. I think it's unneeded here though, I'll remove it.

Reply:
Yes it is: https://github.com/rust-lang/rust/blob/61f5a0092364061ec5649ca98d5e3e9b927880fe/src/tools/compiletest/src/runtest.rs#L1955
You can test by running `x.py test -vv`.